tests/Unit/Visualization/Python/Test_PlotTrajectories.py

```diff
@@ -0,0 +1,143 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+import os
+import shutil
+import unittest
+
+import numpy as np
+from click.testing import CliRunner
+
+import spectre.IO.H5 as spectre_h5
+from spectre.Informer import unit_test_build_path, unit_test_src_path
+from spectre.Visualization.PlotTrajectories import plot_trajectories_command
+
+
+class TestPlotTrajectories(unittest.TestCase):
+    def setUp(self):
+        self.test_dir = os.path.join(
+            unit_test_build_path(), "Visualization", "PlotTrajectories"
+        )
+        self.h5_filename = os.path.join(
+            unit_test_src_path(),
+            "Visualization/Python",
+            "TestPlotTrajectoriesReductions.h5",
+        )
+        os.makedirs(self.test_dir, exist_ok=True)
+        self.create_h5_file_if_not_exists()
+
+    def tearDown(self):
+        if TestPlotTrajectories.delete_test_dir:
```

Comment:
You don't need this `if`. The temporary test directory can always be deleted here.